packages/grid/_modules_/grid/models/api/gridCsvExportApi.ts

```diff
@@ -11,5 +13,5 @@ export interface GridCsvExportApi {
    * Export the grid data as CSV.
    * @returns void
    */
-  exportDataAsCsv: () => void;
+  exportDataAsCsv: (options?: GridExportCsvOptions, fileName?: string) => void;
```

Comment:
I think `fileName` is also an option of `GridExportCsvOptions`.

```suggestion
  exportDataAsCsv: (options?: GridExportCsvOptions) => void;
```

Reply:
I think fileName is a common option for all formats (csv, pdf, excel). The user could set fileName without setting any other options.

Reply:
It would be great to have the same type used for the component prop and the `apiRer.exportDataAsCsv`'s option. Maybe a `<GridToolbarExport csvOptions={}` prop with the `GridExportCsvOptions` type?